install_files/ansible-base/roles/app-test/tasks/main.yml

```diff
@@ -14,3 +14,7 @@
 - include: dev_setup_xvfb_for_functional_tests.yml
 
 - include: disable_mprotect_ff_plugin_container.yml
+
+- include: download_verify_tbb.yml
+
+- include: download_ff_esr.yml
```

Comment:
I'm thinking this tasklist should get merged with `dev_setup_xvfb_for_functional_tests`, which could probably be renamed to something like `install_functional_test_dependencies`. Do also remove the FF 46.0.1 related tasks as that's what these changes are intended to replace.